src/random.cpp

```diff
@@ -336,6 +337,8 @@ struct RNGState {
         memory_cleanse(buf, 64);
         return ret;
     }
+
+    Mutex& GetOpenSSLLock(int i) { return cs_openssl[i]; }
```

Comment:
In commit "Encapsulate RNGState better" (bfd3a23f277c16236ba6ea7391a98277226c1470)

Seems more accurate to call it Mutex than Lock (as in 7939daa351a31a36f2086771611b501418536989), if Mutex is the object that gets locked, and Lock is the object that locks it.

Reply:
Done.